### PR TITLE
Update imports to ipython 4 architecture.

### PR DESCRIPTION
**Pull request by jonathanrocher**
Moved imports from IPython's package to `ipykernel` package.

### CODE REVIEW

**Comment by jonathanrocher** [comment]
@dpinte As we talked about, this is the update to the `ipython_kernel`'s import to `ipython4`.

File: travis-ci-requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 git+http://github.com/enthought/apptools.git#egg=apptools
 git+http://github.com/enthought/traits.git#egg=traits
 git+http://github.com/enthought/traitsui.git#egg=traitsui ; python_version >= "3.0"
-ipython[qtconsole]<4 ; python_version >= "2.7"
+ipython[qtconsole] ; python_version >= "2.7"
```
Comment (dpinte) [inline travis-ci-requirements.txt]: should it be `>=4` ?

**Comment by dpinte** [comment]
@jonathanrocher merging master in this PR should solve the failing tests.

File: travis-ci-requirements.txt (hunk 1 above)
Comment (jonathanrocher) [inline travis-ci-requirements.txt]: Yes, good point. Done.

**Comment by jonathanrocher** [comment]
@dpinte Ready for another look.

**Comment by dpinte** [comment]
@jonathanrocher it is merged!

**Pull request merged**